app/controllers/merchants_controller.rb

```diff
@@ -5,7 +5,11 @@ def index
   end
 
   def show
-    @merchant = Merchant.find(params[:id])
+    if current_admin?
+      redirect_to "/admin/merchants/#{params[:id]}"
+    else
```

Comment:
I think you could use a path helper here if you wanted? Idk if we decided that was the syntax we were going to stick with or not